src/category_theory/limits/preserves/shapes.lean

```diff
@@ -0,0 +1,70 @@
+import category_theory.limits.preserves
```

Comment:
Should this PR move `category_theory.limits.preserves` to `category_theory.limits.preserves.basic`? Do we ever have a Lean file `foo.lean` when we also have a directory `foo/`?

Reply:
Moved!